gulpfile.js

```diff
@@ -356,6 +357,25 @@ gulp.task('watch', function() {
 
 });
 
+/**
+ * Run test once and exit
+ */
+gulp.task('test', function (done) {
+  new Server({
+    configFile: __dirname + '/karma.conf.js',
+    singleRun: true
+  }, done).start();
+});
+
+/**
+ * Watch for file changes and re-run tests on each change
+ */
+gulp.task('test:watch', function (done) {
```

Comment:
Actually we have two types of test switches, one runs only once and another starts on changes like watch so just to make it distinguish as watch it will be good to take it as test:watch. And I don't think it's confusing it's watch for test simple. Isn't it?